solutions/fully-configurable/variables.tf

```diff
@@ -69,16 +69,8 @@ variable "access_tags" {
 
 variable "service_plan" {
```

Comment:
Not mentioned in the PR but you must also update the example accordingly as they have older information.
Refer [here](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/blob/f86018b9d722bb103bc4dae5191068dadc95c6a8/examples/basic/variables.tf#L47).